userguide/subscription/includes/catalog-core.adoc

```diff
@@ -361,11 +355,11 @@ In the remainder of this section we illustrate each type of rule supported by th
 
 === Billing Alignment Rules
 
-This rule uses the Creation Context and configures type of billing alignment used with a subscription. There are three kinds of alignment available:
+This rule uses the Creation Context and configures the type of billing alignment used with a subscription. There are three kinds of alignment available:
 
-* ACCOUNT - this means that the billing cycle of the subscription will be lined up with the bill cycle day of the account (if you didn't specify a value, the system will generate one using the first recurring bill date of all subscriptions with an `ACCOUNT` billing alignment). In some cases, this is undesirable because it means that a proration charge will be applied on first billing to line up the cycles.
+* ACCOUNT - this alignment means that the billing cycle of the subscription will be lined up with the bill cycle day of the account (if you didn't specify a value, the system will generate one using the first recurring bill date of all subscriptions with an `ACCOUNT` billing alignment). In some cases, this is undesirable because it means that a proration charge will be applied on first billing to line up the cycles.
 * SUBSCRIPTION - this alignment will cause the subscriptions bill cycle to line up with the first bill day of the subscription plan. So, if the subscription starts on January 3rd and has a 15 day free trial, the first billed day with be January 18th and the bill cycle day for the subscription will be set to 18.
-* BUNDLE - setting the alignment to bundle is useful for add-ons because it sets the bill cycle day to be aligned with whatever the base plan is using.
+* BUNDLE - this alignment is useful for add-ons because it sets the bill cycle day to be aligned with whatever the base plan is using.
```

Comment:
@sbrossie Closely related to my remark on the other PR: should we not mention "bill cycle day" here? BUNDLE aligned simply means all subscriptions on that bundle are invoiced on the same day.